tests/lib/rules/no-implicit-coercion.js

```diff
@@ -28,6 +28,11 @@ ruleTester.run("no-implicit-coercion", rule, {
         {code: "~foo"},
         {code: "-foo"},
         {code: "2 * foo"},
+        {code: "1 * 1234"},
```

Comment:
I think that can be in `no-useless-expression` or something. This is not a coercion so we shouldn't warn. What do you think?